skbio/stats/ordination/_utils.py

```diff
@@ -196,3 +196,81 @@ def f_matrix(E_matrix):
     col_means = E_matrix.mean(axis=0, keepdims=True)
     matrix_mean = E_matrix.mean()
     return E_matrix - row_means - col_means + matrix_mean
+
+
+def center_distance_matrix_inplace(distance_matrix):
+    """
+    Centers a distance matrix inplace.
+
+    FYI: If the used distance was euclidean, pairwise distances
+    needn't be computed from the data table Y because F_matrix =
+    Y.dot(Y.T) (if Y has been centred).
+    But since we're expecting distance_matrix to be non-euclidian,
+    we do the following computation as per
+    Numerical Ecology (Legendre & Legendre 1998).
+
+    Parameters
+    ----------
+    distance_matrix : 2D array_like
+        Distance matrix.
+    """
+
+    return f_matrix_inplace(e_matrix_inplace(distance_matrix))
+
+
+def e_matrix_inplace(distance_matrix):
+    """
+    Compute E matrix from a distance matrix inplace.
+    Squares and divides by -2 the input element-wise. Eq. 9.20 in
+    Legendre & Legendre 1998.
+
+    Modified from skbio e_matrix function,
```

Comment:
the original `e_matrix` is deleted? suggest deleting this line from docstring too. it's confusing to people without background info.